src/toolbars/examples/toolbar.js

```diff
@@ -117,8 +119,9 @@
           </div>
         </pf-card-view>
       </div>
-      <div class="col-md-12" ng-if="viewType == 'tableView'">
+      <div class="col-md-12" ng-if="viewType == 'tableView' && showComponent">
         <pf-table-view config="tableConfig"
```

Comment:
I'm finding the $onChanges to be not consistent.  It fires as expected when I have the "[ ] Show Pagination" in pfCardView, which sets the $scope.pageConfig accordingly.  However, when same code is in pfToolbar example, setting $scope.pageConfig doesn't trigger an $onChange!!!  Very frustrating!  
The 'showComponent' is a way to quickly add/remove the component from the dom so that the $onInit is called, which correctly sets the $scope.pageConfig.
This would only be an issue when trying to toggle on/off pagination controls, which I'm not sure is a realistic usecase.